sdk/src/builder.rs

```diff
@@ -1495,6 +1495,11 @@ impl Builder {
         W: Write + Read + Seek + Send,
     {
         let settings = crate::settings::get_settings().unwrap_or_default();
+        let http_resolver = if _sync {
```

Comment:
I'd like to explore in a future PR whether the HTTP resolver can/should be configured via Settings. Please add a TO DO to that effect.